ratpack-core/src/main/java/ratpack/server/internal/DefaultRatpackServer.java

```diff
@@ -243,7 +243,11 @@ protected void initChannel(SocketChannel ch) throws Exception {
             pipeline.addLast("ssl", new SslHandler(sslEngine));
           }
 
-          pipeline.addLast("decoder", new HttpRequestDecoder(4096, 8192, serverConfig.getMaxChunkSize(), false));
+          pipeline.addLast("decoder", new HttpRequestDecoder(
+            serverConfig.getMaxInitialLineLength(),
+            serverConfig.getMaxHeaderSize(),
+            serverConfig.getMaxChunkSize(),
+            false));
```

Comment:
Can you pleas move the last parens to the next line to balance the opening parens.